src/App.tsx

```diff
@@ -1,51 +1,93 @@
+import classNames from 'classnames';
 import React from 'react';
 import './App.scss';
+import goods from './goods';
 
-// import goodsFromServer from './goods';
+type State = {
+  selectedGood: string,
+};
 
-export const App: React.FC = () => {
-  return (
-    <main className="App">
-      <header className="App__header">
-        <h1 className="App__title">
-          Jam is selected
-        </h1>
+export class App extends React.Component<{}, State> {
+  state: State = {
+    selectedGood: 'Jam',
+  };
 
-        <button
-          type="button"
-          className="App__clear"
-        >
-          Clear
-        </button>
-      </header>
+  render() {
+    const { selectedGood } = this.state;
 
-      <ul>
-        <li className="Good">Dumplings</li>
-        <li className="Good">Carrot</li>
-        <li className="Good">Eggs</li>
-        <li className="Good">Ice cream</li>
-        <li className="Good">Apple</li>
-        <li className="Good">Bread</li>
-        <li className="Good">Fish</li>
-        <li className="Good">Honey</li>
-        <li className="Good Good--active">Jam</li>
-        <li className="Good">Garlic</li>
-      </ul>
+    return (
+      <main className="App">
+        <header className="App__header">
+          <h1 className="App__title">
+            {selectedGood}
+            {' '}
+            is selected
```

Comment:
Use interpolation instead, it looks more compact
```suggestion
            {`${selectedGood} is selected`}
```